File: api/webAPI/WebApplication1/Models/Poziv.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WebApplication1.Models
{
    public class Poziv
    {
        public int idPoziva { get; set; }
        public string Razlog { get; set; }
        public string Komentar { get; set; }
        public string Kvar { get; set; }
        public int idPotrosaca { get; set; }
    }
}

File: angular-web2-app/src/app/entities/poziv/poziv.ts
export class Poziv {
    idPoziva:number;
    Razlog:string;
    Komentar:string;
    Kvar:string;
    idPotrosaca:number;

    constructor(idPoziva:number, Razlog:string, Komentar:string, Kvar:string, idPotrosaca:number){
        this.idPoziva = idPoziva;
        this.Razlog = Razlog;
        this.Komentar = Komentar;
        this.Kvar = Kvar;
        this.idPotrosaca = idPotrosaca;
    }
}